### DISCUSSION TITLE
example-app-router-playground does not work when basePath is set

### DISCUSSION THREAD

**Discussion by andeksan** [amannn/next-intl#1249] [General]
I am trying to implement similar stuff what in example-app-router-playground, but the example does not work as soon as basePath: '/baseurl',  is set in next.config.mjs. 
when http://localhost:3000/baseurl is, i get 404 page.  if http://localhost:3000/baseurl/de, the page is rendered with errors.
I am interesting i am doing something wrong or the project requires an additional configuration?

**Comment by amannn** [comment]
`basePath` should be automatically handled when it's used. The only thing I noticed while testing with this option, is that it's important to include `/` in your matcher (see the docs for [`basePath`](https://next-intl-docs.vercel.app/docs/routing/middleware#base-path)).

**Comment by andeksan** [comment]
Thanks a lot, your advice solves my issue.
